tasktracker/static/tasktracker/index.js

```diff
@@ -77,4 +77,6 @@ const deleteTask = async (task) => {
   res = await fetch(`tasks/${task.id}`, {
     method: "DELETE",
   });
+  loadTasks()
+  /* bu fonksiyonu neden çağıramıyorum */
```

Comment:
You can call this function from here. Have you checked console logs maybe there is another issue?

Chrome - f12 -  console

You can also add `debugger` to the rows that you would like to check whether they have called or not.

Reply:
Actually, what I want is to load the Tasks after one task deleted. For the current system, when I clicked the delete button on a task, it deletes from the database. But it still requires a refresh to update the frontend.

Reply:
Okey, I found the bug. I was adding reloaded tasks before deleting the innerHTML of the overall page. Now, it works okey